Add blackbox_exporter docs. Update changelog. (#185)
CrunchyData/pgmonitor: hugo/content/exporter/_index.md

| postgres/queries_*.yml | `/etc/postgres_exporter/##/queries_*.yml`  |
| postgres/pgbackrest-info.sh | `/usr/bin/pgbackrest-info.sh` |

##### blackbox_exporter

The following pgMonitor configuration files should be placed according to the following mapping:

| pgMonitor Configuration File | System Location |
|------------------------------|-----------------|

> | blackbox/blackbox_exporter.sysconfig  | `/etc/sysconfig/blackbox_exporter`   |

Reviewer: ```suggestion
| blackbox/blackbox_exporter.sysconfig  | {{ shell }}/etc/sysconfig/blackbox_exporter{{ /shell }}   |
```

Author: Actually, I don't think these submodule changes have been applied to the pgmonitor repository. I made sure it was all up to date, but when I run hugo, these formatting flags aren't being applied

> | blackbox/crunchy-blackbox.yml| `/etc/blackbox_exporter/crunchy-blackbox.yml` |

Reviewer: ```suggestion
| blackbox/crunchy-blackbox.yml| {{ shell }}/etc/blackbox_exporter/crunchy-blackbox.yml{{ /shell }} |
```

The script requires being run by a database superuser by default since it must be able to run a scan on every table. If you'd like to not run it as a superuser, you will have to create a new role that has read permissions on all tables in all schemas that are to be monitored for bloat. You can then change the user in the connection string option to the script.

##### Blackbox Exporter

> The configuration file for the blackbox_exporter provided by pgMonitor (`/etc/blackbox_exporter/crunchy-blackbox.yml`) provides a probe for monitoring any ivp4 tcp port status. The actual target and port being monitored is controlled via the Prometheus target configuration system. Please see the pgMonitor Prometheus documentation for further details. If any additional blackbox probes are desired, please see the upstream documentation.

Reviewer: ```suggestion
The configuration file for the blackbox_exporter provided by pgMonitor ({{ shell }}/etc/blackbox_exporter/crunchy-blackbox.yml{{ /shell }}) provides a probe for monitoring any IPv4 TCP port status. The actual target and port being monitored are controlled via the Prometheus target configuration system. Please see the pgMonitor Prometheus documentation for further details. If any additional Blackbox probes are desired, please see the upstream documentation.
```

Author: Agree here, this can be made capital
